crates/runtime/src/http/routes.rs

```diff
@@ -74,7 +74,9 @@ pub(crate) fn routes(
             .route("/v1/embeddings", post(v1::embeddings::post))
             .route("/v1/search", post(v1::search::post))
             .route("/v1/tools", get(v1::tools::list))
+            .route("/v1/tools/:name", post(v1::tools::post))
             .route("/v1/tool/:name", post(v1::tools::post))
```

Comment:
We should add a comment this route is deprecated (so future people that come across this don't wonder why we have two, and which one is the correct one)